test/functional/p2p_invalid_messages.py

```diff
@@ -101,11 +101,10 @@ def run_test(self):
             msg_over_size = msg_unrecognized(str_data="b" * (valid_data_limit + 1))
             assert len(msg_over_size.serialize()) == (msg_limit + 1)
 
-            with node.assert_debug_log(["Oversized message from peer=4, disconnecting"]):
```

Comment:
In commit "Remove oversized message detection from log and interface" (6a91499496d76c2b3e84489e9723b60514fb08db)

Is there a different log message this could check for? It would seem better if the test could continue checking the reason for disconnect.